packages/salesforcedx-vscode-soql/test/ui-test/soql-ui-smoke.test.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as path from 'path';
+import {
+  By,
+  EditorView,
+  InputBox,
+  TextEditor,
+  VSBrowser,
+  WebDriver,
+  WebView,
+  Workbench
+} from 'vscode-extension-tester';
+
+// tslint:disable-next-line:only-arrow-functions
+describe('In project folder, SOQL files should', function() {
+  this.timeout(55000);
+  let browser: VSBrowser;
+  let driver: WebDriver;
+  const folderPath =
+    path.resolve(
+      __dirname,
+      '..',
+      '..',
+      '..',
+      'test',
+      'ui-test',
+      'resources',
+      'sfdx-test-project'
+    ) + path.sep;
+  const filename = 'example.soql';
+
+  const openFolder = async (folder: string) => {
+    const workbench = new Workbench();
+    const cmdTitle = 'Extest: Open Folder';
```

Comment:
`extest` is a utility command provided by the testing framework. There are some recommended ways to open files and folders but they have been unreliable for us. 
Using the `extest` command has been simple and stable so far. 
https://github.com/redhat-developer/vscode-extension-tester/wiki#opening-files-and-folders